### PR TITLE
fs: add append method

### PR DESCRIPTION
**Pull request by aborah-sudo**
Add append support for fs module

### CODE REVIEW

File: pytest_mh/utils/fs.py [hunk 1]
```diff
@@ -207,6 +207,39 @@ def write(
...
+            contents = textwrap.dedent(contents).strip()
+
+        self.backup(path)
+        self.logger.info(f'Writing file "{path}" on {self.host.hostname}', extra={"data": {"Contents": contents}})
+
+        self.host.ssh.run(
+            f"""
+                set -ex
+                cat >> '{path}'
+                {self.__gen_chattrs(path)}
```
Comment (pbrezina) [inline pytest_mh/utils/fs.py:237]: Since we do not set mode, user and  group we do not need this line.

File: pytest_mh/utils/fs.py [hunk 2]
```diff
@@ -207,6 +207,39 @@ def write(
...
+        :param contents: File contents to write.
+        :type contents: str
+        :param dedent: Automatically dedent and strip file contents, defaults to True
+        :type dedent: bool, optional
+        """
+        if dedent:
+            contents = textwrap.dedent(contents).strip()
+
+        self.backup(path)
+        self.logger.info(f'Writing file "{path}" on {self.host.hostname}', extra={"data": {"Contents": contents}})
```
Comment (pbrezina) [inline pytest_mh/utils/fs.py:231]: Appending to file ...?

**Review by pbrezina — CHANGES_REQUESTED** [review]
Thank you, see small comment inside.

Also please change the commit message to "fs: add append method"

File: pytest_mh/utils/fs.py (hunk 2 above)
Comment (aborah-sudo) [reply pytest_mh/utils/fs.py:231]: Fixed

File: pytest_mh/utils/fs.py (hunk 1 above)
Comment (aborah-sudo) [reply pytest_mh/utils/fs.py:237]: Fixed

**Review by pbrezina — APPROVED** [review]

**Pull request merged**